censys/cli/commands/asm.py

```diff
@@ -557,6 +557,13 @@ def cli_execute_saved_query_by_name(args: argparse.Namespace):
         sys.exit(1)
     query = results[0]["query"]
 
+    if args.page_size > 1000:
```

Comment:
Hey, this looks like a great addition, maybe we can add this argument condition to before we fetch the saved query?